website/src/contribute/ptransform-style-guide.md

```diff
@@ -30,7 +30,7 @@ _A style guide for writers of new reusable PTransforms._
 ### Consistency
 Be consistent with prior art:
 
-* Adhere to [Beam design principles]({{ site.baseurl }}/contribute/design-principles/).
+* Adhere to [Beam design principles]({{ site.baseurl }}/contribute/).
```

Comment:
Should we adjust the title, e.g. `Please read the contribution guide`?

Reply:
Sure, I'll change the title.